# Generated by Django 2.0 on 2017-12-07 13:31

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='Outage',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('Prompt_Name', models.TextField(help_text='Enter the corrosponding OutageManagement prompt file name ', max_length=1000)),
                ('Service', models.CharField(help_text='Enter Outage Servcie Name', max_length=200)),
                ('Start_Date', models.DateTimeField(null=True)),
                ('End_Date', models.DateTimeField(null=True)),
                ('Enable_Flag', models.BooleanField()),
                ('Created_By', models.OneToOneField(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='Outage_Service',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('Service', models.CharField(blank=True, max_length=60)),
                ('Outage_ID', models.CharField(blank=True, max_length=100)),
            ],
        ),
        migrations.AddField(
            model_name='outage',
            name='Outage_ID',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='OutageManagement.Outage_Service'),
        ),
    ]
